app/Providers/AppServiceProvider.php

```diff
@@ -11,7 +14,9 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register(): void
     {
-        //
+        $this->app->bind(CreateAirlineAction::class, function () {
+            return new CreateAirlineAction();
+        });
```

Comment:
```suggestion
        //
```
This code is nice, but it needs to be used correctly. Binding is helpful for telling the framework how an interface should be resolved (meaning, an instance of which class should be provided when injecting that interface as a dependency). In this case, `CreateAirlineAction` isn't an interface but a class, so you don't need to tell the framework how to resolve it.